### PR TITLE
cleanup dependencies

### PR DESCRIPTION
**Pull request by lukebrody**
- remove unused/unnecessary react packages
- organize react packages into dev and not dev dependencies
- add python dependencies

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -17,23 +16,26 @@
...
+    "webpack": "^5.88.2",
     "@babel/core": "^7.22.9",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
+    "babel-loader": "^9.1.0",
+    "downloads-folder": "^3.0.3"
+  },
+  "dependencies": {
```
Comment (kavigupta) [inline react/package.json:33]: Why are there two dependencies dicts

File: react/test/article_test.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Selector, ClientFunction } from 'testcafe';
 
-const TARGET = "http://teroknor:8000"
+const TARGET = "http://localhost:8000"
```
Comment (kavigupta) [inline react/test/article_test.js:3]: Making this an argument would be ideal

File: react/package.json (hunk 1 above)
Comment (lukebrody) [reply react/package.json:33]: One is devDependencies
> "dependencies": Packages required by your application in production.
> "devDependencies": Packages that are only needed for local development and testing.

Comment (kavigupta) [reply react/package.json:33]: No like you have devdependencies and then two dependencies keys

Comment (kavigupta) [reply react/package.json:33]: Oh wait I misinterpreted the diff

File: react/test/article_test.js (hunk 2 above)
Comment (lukebrody) [reply react/test/article_test.js]: added env var?

**Review by kavigupta — APPROVED** [review]

**Pull request merged**